Makefile

```diff
@@ -7,7 +7,11 @@ TAGS := daemon
 endif
 
 ifndef PKGS
-PKGS := $(shell go list ./... 2>&1 | grep -v 'github.com/libopenstorage/stork/vendor')
+PKGS := $(shell go list ./... 2>&1 | grep -v 'github.com/libopenstorage/stora/vendor')
+endif
+
+ifndef GOVET_PKGS
+GOVET_PKGS=$(shell  go list ./... | grep -v vendor | grep -v pkg/client/informers/externalversions | grep -v versioned | grep -v pkg/apis/stork.com)
```

Comment:
Just add the exceptions in PKGS instead of defining GOVET_PKGS

Reply:
Done